web/admin/src/Modal.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react'
+import './App.css'
+import Modal  from 'react-modal'
+
+export class CustomModal extends Component {
```

Comment:
We are using 2 spaces for tabs elsewhere.  Please update tab settings to use 2 spaces.